api/app/src/command/medical/document/document-query.ts

```diff
@@ -50,21 +49,11 @@ export async function queryDocumentsAcrossHIEs({
   const updatedPatient = await updateDocQuery({
     patient: { id: patient.id, cxId: patient.cxId },
     downloadProgress: { status: "processing" },
-    restart: true,
+    reset: true,
   });
 
   // intentionally asynchronous, not waiting for the result
-  getDocumentsFromCW({ patient, facilityId, override })
-    .then((amountProcessed: number) => {
-      log(`Finished processing ${amountProcessed} documents.`);
-    })
-    .catch(err => {
-      updateDocQuery({
-        patient: { id: patient.id, cxId: patient.cxId },
-        downloadProgress: { status: "failed" },
-      });
-      processAsyncError(`doc.list.getDocumentsFromCW`)(err);
-    });
+  getDocumentsFromCW({ patient, facilityId, override });
```

Comment:
why are we removing the error handling here?

Reply:
forgot to mention: we already have that on the actual function, this was triggering two notifications to Sentry.